src/src/components/form/CustomSection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <th
+    colspan="4"
+    class="sectionHeader"
+  >
+    <h3
+      @click="toggle($event)"
+      v-html="field.title"
+      :class="{'toggle' : field.canToggleSection}"
+    ></h3>
+    <span>
+        <a
+          v-if="field.canDeleteSection"
+          href="" 
+          @click="removeFields($event)"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 10 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        @click="removeFields($event)"
```